app/templates/userhome.html

```diff
@@ -61,8 +66,19 @@ <h2 class="header-margin">Profile</h2>
       {% endif %}
     </div>
 
+    {# TODO: Replace hardcoding in the line below #}
+    {% if not profile.description and not profile.badges.all and not links.all and not profile.tags.all %}
+    <div class="col-12">
+      {% if profile != request.user.profile %}
+      <p>{{ profile.username }} has not given any user details.</p>
+      {% else %}
+      <p>No user details! Go to <a href="{% url 'edit_profile' %}">Edit Profile</a> to add something about yourself.</p>
+      {% endif %}
+    </div>
+    {% else %}
```

Comment:
I had hardcoded the if statement on line 70, it should be replaced.

Reply:
I think it's alright tbh.